interfaces/kubernetes_manifest/v0/schema.py

```diff
@@ -0,0 +1,49 @@
+"""This file defines the schemas for the provider and requirer sides of the kubernetes_manifest interface.
+
+Examples:
+    RequirerSchema:
+        unit: <empty>
+        app:
+            kubernetes_manifests: [
+                                        {
+                                            "apiVersion": "v1",
+                                            "kind": "Secret",
+                                            "metadata": {
+                                                "name": "seldon-rclone-secret",
+                                                "labels": {"user.kubeflow.org/enabled": "true"},
+                                            },
+                                            "stringData": {"RCLONE_CONFIG_MYS3_TYPE": "test"},
+                                        },
+                                        {
+                                            "apiVersion": "v1",
+                                            "kind": "Secret",
+                                            "metadata": {
+                                                "name": "mlpipeline-minio-artifact",
+                                                "labels": {"user.kubeflow.org/enabled": "true"},
+                                            },
+                                            "stringData": {"AWS_ACCESS_KEY_ID": "access_key"},
+                                        },
+                                    ]
+"""
+
+from typing import List
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel
+
+
+class KubernetesManifest(BaseModel):
+    """
+    Representation of a Kubernetes Object sent to Kubernetes Manifests.
+
+    Args:
+        manifest_content: the content of the Kubernetes manifest file
+    """
+
+    manifest_content: str
```

Comment:
FYI, you can put the field description in... the field description!

```suggestion
    manifest_content: str = pydantic.Field(..., description="The content of the kubernetes manifest file.")
```

as it's technically not an arg.